test/FastTests/Issues/RavenBooleanQueryTests.cs

```diff
@@ -11,7 +12,7 @@ namespace FastTests.Issues;
 //Low-level query optimization tests. 
 public class RavenBooleanQueryTests : RavenTestBase
 {
-    [Theory]
+    [RavenTheory(RavenTestCategory.Core)]
```

Comment:
shouldn't this be Querying?

Reply:
The comment throw it off. Because low-level stuff is either Core or Corax.